db/schema.rb

```diff
@@ -35,7 +35,7 @@
   add_index "builds", ["project_id"], name: "index_builds_on_project_id", using: :btree
 
   create_table "commands", force: :cascade do |t|
-    t.text     "command",    limit: 10485760
+    t.text     "command",    limit: 16777215
```

Comment:
we changed it to 10485760 because anything bigger blew up on travis ... if
you get it green then that's fine ;)

On Fri, Aug 19, 2016 at 7:40 AM, Craig Day <notifications@github.com> wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/1227#discussion_r75493344>:
>
> > @@ -35,7 +35,7 @@
> >    add_index "builds", ["project_id"], name: "index_builds_on_project_id", using: :btree
> >
> >    create_table "commands", force: :cascade do |t|
> > -    t.text     "command",    limit: 10485760
> > +    t.text     "command",    limit: 16777215
>
> I'm not sure what the previous value was, but this is 2^24 which is what a
> MySQL MEDIUMTEXT column gets set to.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1227/files/e4b9d7dc970e6ebfa93f2e3f21c6c5ec4b74cefb#r75493344>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyv2PEYMnw5_JfIfcklcr_aMJCw-ks5qhcB6gaJpZM4Jn-LO>
> .
>